.buildkite/pipeline.yml

```diff
@@ -47,75 +47,24 @@ steps:
     env:
       BUILD_RN_WITH_LATEST_NATIVES: ${BUILD_RN_WITH_LATEST_NATIVES}
 
-
-  #
-  # Trigger individual pipelines
-  #
-  - label: 'Trigger Browser pipeline'
-    if: build.env("BUILD_RN_WITH_LATEST_NATIVES") != "true"
-    depends_on: 'package-js'
-    trigger: 'bugsnag-js-browser'
-    build:
-      branch: '${BUILDKITE_BRANCH}'
-      commit: '${BUILDKITE_COMMIT}'
-      message: '${BUILDKITE_MESSAGE}'
-
-  - label: 'Trigger Electron pipeline'
-    if: build.env("BUILD_RN_WITH_LATEST_NATIVES") != "true"
-    trigger: 'bugsnag-js-electron'
-    build:
-      branch: '${BUILDKITE_BRANCH}'
-      commit: '${BUILDKITE_COMMIT}'
-      message: '${BUILDKITE_MESSAGE}'
-
-  - label: 'Trigger Node pipeline'
+  - label: "Trigger Expo pipeline"
     if: build.env("BUILD_RN_WITH_LATEST_NATIVES") != "true"
-    depends_on: 'package-js'
-    trigger: 'bugsnag-js-node'
-    build:
-      branch: '${BUILDKITE_BRANCH}'
-      commit: '${BUILDKITE_COMMIT}'
-      message: '${BUILDKITE_MESSAGE}'
-
-  - label: 'Trigger Expo pipeline'
-    if: build.env("BUILD_RN_WITH_LATEST_NATIVES") != "true"
-    depends_on: 'publish-js'
-    trigger: 'bugsnag-expo'
+    depends_on: "publish-js"
+    trigger: "bugsnag-expo"
     build:
       env:
-        BUGSNAG_JS_BRANCH: '${BUILDKITE_BRANCH}'
-        BUGSNAG_JS_COMMIT: '${BUILDKITE_COMMIT}'
-        # a branch name that's safe to use as a docker cache identifier
-        BUGSNAG_JS_CACHE_SAFE_BRANCH_NAME: '${BRANCH_NAME}'
+        BUGSNAG_JS_BRANCH: "${BUILDKITE_BRANCH}"
+        BUGSNAG_JS_COMMIT: "${BUILDKITE_COMMIT}"
+        # a branch name that"s safe to use as a docker cache identifier
+        BUGSNAG_JS_CACHE_SAFE_BRANCH_NAME: "${BRANCH_NAME}"
 
-  - label: 'Trigger React Native pipeline'
-    depends_on:
-      - 'publish-js'
-      - 'android-builder-base'
-    trigger: 'bugsnag-js-react-native'
-    build:
-      branch: '${BUILDKITE_BRANCH}'
-      commit: '${BUILDKITE_COMMIT}'
-      message: '${BUILDKITE_MESSAGE}'
-
-  - label: 'Trigger React Native CLI pipeline'
-    if: build.env("BUILD_RN_WITH_LATEST_NATIVES") != "true"
-    depends_on:
-      - 'publish-js'
-      - 'android-builder-base'
-    trigger: 'bugsnag-js-react-native-cli'
-    build:
-      branch: '${BUILDKITE_BRANCH}'
-      commit: '${BUILDKITE_COMMIT}'
-      message: '${BUILDKITE_MESSAGE}'
-
-  - label: ':aws-lambda: AWS Lambda tests'
+  - label: ":aws-lambda: AWS Lambda tests"
     if: build.env("BUILD_RN_WITH_LATEST_NATIVES") != "true"
     timeout_in_minutes: 35
     agents:
-      queue: 'opensource-mac-cocoa-10.15'
+      queue: "opensource-mac-cocoa-10.15"
     commands:
-      # force the NPM registry as the default on CI is artifactory, which can't
+      # force the NPM registry as the default on CI is artifactory, which can"t
```

Comment:
slightly aggressive find and replace ;)

Reply:
Ooh, I did realise that was a possibility and tried to catch that when looking over the resulting diff!